.appveyor.yml

```diff
@@ -0,0 +1,20 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+environment:
+  matrix:
+  - TOXENV: py35
+  - TOXENV: py36
+  - TOXENV: py37
```

Comment:
Is there no Python 3.8 there?

Also, this duplicates information from `tox.ini`, is this necessary?